media/src/androidMain/kotlin/DefaultMediaManager.kt

```diff
@@ -82,16 +85,25 @@ private fun AndroidTrackInfo.asTrackInfo(identifier: Int): TrackInfo = TrackInfo
  * Default implementation of [BaseMediaManager]
  * @param mediaSurfaceProvider a [MediaSurfaceProvider] that will automatically call [renderVideoOnSurface] for the latest [MediaSurface]
  * @param coroutineContext the [CoroutineContext] on which the media will be managed
+ * @param context the [Context] to access local resources
  */
-actual class DefaultMediaManager(mediaSurfaceProvider: MediaSurfaceProvider?, coroutineContext: CoroutineContext) : BaseMediaManager(mediaSurfaceProvider, coroutineContext) {
+actual class DefaultMediaManager(mediaSurfaceProvider: MediaSurfaceProvider?, coroutineContext: CoroutineContext, private val context: Context) :
+    BaseMediaManager(mediaSurfaceProvider, coroutineContext) {
 
     /**
      * Builder for creating a [DefaultMediaManager]
      */
     class Builder : BaseMediaManager.Builder {
-        override fun create(mediaSurfaceProvider: MediaSurfaceProvider?, coroutineContext: CoroutineContext): DefaultMediaManager = DefaultMediaManager(
+
+        /** Creates [DefaultMediaManager] with the application context */
+        override fun create(mediaSurfaceProvider: MediaSurfaceProvider?, coroutineContext: CoroutineContext): DefaultMediaManager =
+            create(mediaSurfaceProvider, coroutineContext, ApplicationHolder.applicationContext)
+
+        /** Creates [DefaultMediaManager] */
+        fun create(mediaSurfaceProvider: MediaSurfaceProvider?, coroutineContext: CoroutineContext, context: Context): DefaultMediaManager = DefaultMediaManager(
```

Comment:
Pattern we usually have is providing the context in the constructor of Builder. As it's only used for MediaSource.Bundle though, Id suggest the context should go in there